src/functions.ts

```diff
@@ -62,7 +78,27 @@ export interface CloudFunctionsContext {
   /**
    * The resource that emitted the event.
    */
-  resource?: string;
+  resource?: string | CloudFunctionsResource;
+}
+
+/**
+ * The Cloud Functions resource object for the event.
+ *
+ * @link https://cloud.google.com/functions/docs/writing/background#function_parameters
+ */
+export interface CloudFunctionsResource {
```

Comment:
- I'm not seeing where this is defined.
- I think this only applies to background functions?

Reply:
Yes, this only applies to background functions. It's defined in [here](https://cloud.google.com/functions/docs/writing/background#function_parameters) under `context.resource` as "The resource that emitted the event.".